cloudbuild/cloudbuild.yaml

```diff
@@ -1,16 +1,24 @@
 steps:
 
+- id: Fetch service account key
+  name: gcr.io/cloud-builders/gsutil
+  args:
+  - cp
+  - gs://cloud-marketplace-ops-test-kokoro/application_credentials
+  - /builder/home/service_account.json
+
 - id: Fetch key for the github app
```

Comment:
just a note here: these should no longer be needed since we're public now @vcanaa